__tests__/endpoints.test.js

```diff
@@ -88,5 +88,38 @@ describe('/api/articles', () => {
         })
     })
 })
+describe('/api/articles/:article_id/comments', () => {
+    test('200 with all the comment in ascending order related to the article.article_id', () => {
+        return request(app).get('/api/articles/3/comments').expect(200)
+        .then((res) => {
+            const comments = res.body
+            expect(comments.length).toBe(2)
+            expect(comments).toBeSortedBy('created_at', { ascending: true })
+            comments.forEach((comment) => {
+                expect(typeof comment.comment_id).toBe('number')
+                expect(typeof comment.votes).toBe('number')
+                expect(typeof comment.created_at).toBe('string')
+                expect(typeof comment.author).toBe('string')
+                expect(typeof comment.body).toBe('string')
+                expect(typeof comment.article_id).toBe('number')
```

Comment:
You can be more specific here - would you expect the article id to be any number?